src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
+#include <GSS/GSS.h>
+#else
+#include <gssapi/gssapi_ext.h>
+#include <gssapi/gssapi_krb5.h>
+#endif
```

Comment:
As we have seen when working on Heimdal NTLM prototype, the same files can be available under some heimdal folder. So it is best to disambiguate by adding gssapi_ext.h and gssapi_krb5.h without gssapi and instead modifying CMakeLists to include the folders /usr/include/mit-krb5 and /usr/include/mit-krb5/gssapi